### ISSUE TITLE
1.6.5 Reflection is broken

### ISSUE THREAD

**Issue by MartenM** [upperlevel/book-api#12]
Currently the plugin does not support 1.16.5 due to changes in the NMS code which breaks the reflection.

```xyz.upperlevel.spigot.book.NmsBookHelper$UnsupportedVersionException: Error while executing reflections, submit to developers the following log (version: v1_16_R3)
	at xyz.upperlevel.spigot.book.NmsBookHelper.setPages(NmsBookHelper.java:145) ~[?:?]
	at xyz.upperlevel.spigot.book.BookUtil$BookBuilder.pages(BookUtil.java:136) ~[?:?]
	at nl.martenm.surveyplus.survey.BookSurvey.welcomePlayer(BookSurvey.java:51) ~[?:?]
	at nl.martenm.surveyplus.objects.surveySessions.SurveySession$1.run(SurveySession.java:58) ~[?:?]
	at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftTask.run(CraftTask.java:81) ~[spigot-1.16.5.jar:git-Spigot-5bde311-0bad58f]
	at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:400) ~[spigot-1.16.5.jar:git-Spigot-5bde311-0bad58f]
	at net.minecraft.server.v1_16_R3.MinecraftServer.b(MinecraftServer.java:1060) ~[spigot-1.16.5.jar:git-Spigot-5bde311-0bad58f]
	at net.minecraft.server.v1_16_R3.DedicatedServer.b(DedicatedServer.java:355) ~[spigot-1.16.5.jar:git-Spigot-5bde311-0bad58f]
	at net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1008) ~[spigot-1.16.5.jar:git-Spigot-5bde311-0bad58f]
	at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:847) ~[spigot-1.16.5.jar:git-Spigot-5bde311-0bad58f]
	at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$0(MinecraftServer.java:164) ~[spigot-1.16.5.jar:git-Spigot-5bde311-0bad58f]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_261]
Caused by: java.lang.NullPointerException
	at xyz.upperlevel.spigot.book.NmsBookHelper.setPages(NmsBookHelper.java:139) ~[?:?]
	... 11 more
```
	
I will fork the project and attempt to fix the issue and PR.

**Comment by SnowyCoder** [comment]
What library version are you using? It seems like #9  that was fixed in v1.4

**Comment by MartenM** [comment]
> What library version are you using? It seems like #9 that was fixed in v1.4

Yeah closed the issue. Did not notice the version bump.
Updating to 1.4 fixed the issue.
